web.go

```diff
@@ -807,3 +809,12 @@ func handleSetup(c *gin.Context) {
 
 	c.JSON(http.StatusOK, gin.H{"message": "Device setup completed successfully"})
 }
+
+func handleSendWOLMagicPacket(c *gin.Context) {
+	macAddr := c.Param("mac-addr")
```

Comment:
Would prefer there be some validation on this raw path parameter (is this a reasonable MAC address) before calling the WOL.